core/client/fs/src/main/java/alluxio/client/file/AlluxioFileInStream.java

```diff
@@ -461,4 +464,14 @@ private void handleRetryableException(BlockInStream stream, IOException e) {
     // TODO(lu) consider recovering failed workers
     mFailedWorkers.put(workerAddress, System.currentTimeMillis());
   }
+
+  @Override
+  public void unbuffer() {
```

Comment:
Please, I want to know how and when does the application call the unbuffer method?

Reply:
thanks, I will check it